pkg/controller/workload/workload_processor.go

```diff
@@ -592,6 +592,11 @@ func (p *Processor) handleService(service *workloadapi.Service) error {
 			log.Errorf("storeServiceFrontendData failed, err:%s", err)
 			return err
 		}
+		// get endpoint from ServiceCache, and update service and endpoint map
+		if err = p.storeServiceData(serviceName, service.GetWaypoint(), service.GetPorts()); err != nil {
```

Comment:
This is not right, you can test it after waypoint deleted, we can see some weired error